app/services/ApiJsonMessage.scala

```diff
@@ -0,0 +1,8 @@
+package services
+
+class ApiJsonMessage {
+
+  def create(message: String): String = {
+    "{\"message\": \"" + message + "\"}"
```

Comment:
@Bukarachka "message" is a strange key, let's use "error" instead. As it is basically error message